index.html

```diff
@@ -136,6 +136,25 @@ <h1 id="title">Top Github Issues</h1>
     }).join('\n');
 }
 
+function getAssigneesSortKey(issue) {
+    const assignees = issue.assignees.sort().map(assignee => assignee.login.toLowerCase());
+    if (assignees.length === 0) {
+        return '\ue83a'; // force unassigned to the bottom of the sort
+    }
+    if (githubUsername && assignees.indexOf(githubUsername.toLowerCase()) >= 0) {
+        return '!'; // force user's own issues to top of the sort
+    }
+    return assignees.join(',');
+}
```

Comment:
I'm erring on the side of not including this sorting logic. We may soon move away from users providing their usernames and personal access tokens.